src/aggregator/server/rawtcp/options.go

```diff
@@ -34,7 +33,7 @@ const (
 	defaultErrorLogLimitPerSecond = 0
 
 	// The default read buffer size for raw TCP connections.
-	defaultReadBufferSize = 1440
+	defaultReadBufferSize = 65536
```

Comment:
note: this may be something we want to export eventually versus conventionally setting in N different places